docs/source/install/bwc.rst

```diff
@@ -0,0 +1,33 @@
+Installing |bwc|
+================
+
+StackStorm is an event-driven DevOps automation platform with all the essential features suitable for small businesses and teams. It’s free and open source under the Apache 2.0 license.
```

Comment:
we don't have substitution for "StackStorm" :( yet. |st2| resolves to BWC if `-t enterprise`